src/models/User.js

```diff
@@ -1,32 +1,80 @@
-const { Schema, model } = require('mongoose');
+const {
+  Schema,
+  model,
+  Types: { ObjectId },
+} = require("mongoose");
+
 const emailRegex = /^[a-zA-Z0-9._%+-]{3,}@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
 // const emailRegex2 = /^[\w-]+(\.[\w-]+)*@[\w-]+(\.[\w-]+)+$/;
 
 //TODO: add validation for fields in the schema
 //TODO: add city and likedEvents
 const userSchema = new Schema({
-    username: { type: String, require: true, },
-    email: {
-        type: String, required: true, unique: true, validate: {
-            validator: (value) => emailRegex.test(value),
-            message: "Invalid email",
-
-        }
+  email: {
+    type: String,
+    required: true,
+    unique: true,
```

Comment:
Is this working?

Reply:
I'm not sure what you talking about but return the propper response.

Reply:
I mean if you try to add two identical emails, is it working?